src/server/parser.js

```diff
@@ -0,0 +1,317 @@
+/*jslint devel: true, nomen: true, plusplus: true, regexp: true, indent: 2, maxlen: 100 */
+
+"use strict";
+
+var logger = require('./logger');
+var async = require('async');
```

Comment:
It turns out that `async` alone is not a new reserved word, the phrase `async function` is reserved, probably because `async` is so commonly used as a variable. `await` is a new reserved word.